gatsby-template/package.json

```diff
@@ -49,7 +50,8 @@
     "lint-staged": "^9.2.1",
     "pre-commit": "^1.2.2",
     "prettier": "^1.17.0",
-    "wait-for-netlify-preview": "^1.2.1"
+    "wait-for-netlify-preview": "^1.2.1",
+    "to-style": "^1.3.3"
```

Comment:
why do you need those dependencies?

Reply:
accidentally still exist. 
I have been added for tackling  [#60 ](https://github.com/mathieudutour/medium-to-own-blog/issues/60)
but now it's fixed in **version 0.3.0**

remove in next commit